frontend/src/features/admin/edit-user/EditUserPage.scss

```diff
@@ -1,41 +1,41 @@
 @import '../../../variables';
-.userInfo {
-  width: 40%;
-  vertical-align: middle;
-  text-align: left;
-  display: table;
-}
+.user-edit-form-container {
```

Comment:
this scss was applying to the app globally since this isn't a css module. Manually only applying these rules to children of a certain class prevents this.